src/Nucleation.jl

```diff
@@ -0,0 +1,94 @@
+module Nucleation
+
+"""
+    nucleation_timestep(rh, temp, so4)
+
+ - `rh` - Relative humidity, expressed as a percentage between 0 and 1
+ - `temp` - Temperature (K)
+ - `so4` - Total gas phase concentration of sulfuric acid (cm^-3)
+
+Returns the nucleation rate for a single timestep under the given conditions.
+All newly formed particles will be placed into the Aitken mode.
```

Comment:
Would be good to say (probably in the docs) that we are talking here about the sulfuric acid only